src/views/questionView.js

```diff
@@ -19,6 +20,7 @@ export const createQuestionElement = (
 
   // I use String.raw just to get fancy colors for the HTML in VS Code.
   element.innerHTML = String.raw`
+  <p>Question : ${index + 1} : 10</p>
```

Comment:
I suggest using variables instead of manually stated values
```suggestion
  <p>Question : ${index + 1} : ${maxQuestions}</p>
```